packages/thirdParty/components/Loader/index.jsx

```diff
@@ -2,9 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import RouteMonitor from '../RouteMonitor';
+import DriftWidget from '../DriftWidget';
 
-const Loader = ({ appCues, location }) => (
-  <RouteMonitor appCues={appCues} pathname={location.pathname} />
+const Loader = ({ appCues, location, drift }) => (
+  <>
+    <RouteMonitor appCues={appCues} pathname={location.pathname} />
+    {drift.ready && <DriftWidget id={drift.id} email={drift.email} />}
```

Comment:
For safety I'd probably use optional chaining here 🙂 
```suggestion
    {drift?.ready && <DriftWidget id={drift.id} email={drift.email} />}
```